OMEdit/OMEditGUI/Editors/MetaModelEditor.cpp

```diff
@@ -434,6 +434,9 @@ void MetaModelEditor::addInterfacesData(QDomElement interfaces)
   QDomNodeList subModelList = mXmlDocument.elementsByTagName("SubModel");
   for (int i = 0 ; i < subModelList.size() ; i++) {
     QDomElement subModel = subModelList.at(i).toElement();
+    while(!subModel.firstChildElement("InterfacePoint").isNull()) { //First remove all existing interfaces!
```

Comment:
But what if the user removes an interface from the external model, should it not disappear in the meta model when running fetch interfaces?